Modules/@babylonjs/react-native-iosandroid/.gitignore

```diff
@@ -0,0 +1,51 @@
+# OSX
```

Comment:
Shouldn't this file mostly be a move (from react-native)?

Reply:
No, it's added and same from package-lock.json: a new `react-native-iosandroid` folder is added with package-lock and gitignore. That's why there is a 22K lines more.